src/Library/demos/Navigation View/main.blp

```diff
@@ -0,0 +1,90 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  Box {
+    halign: center;
+    valign: start;
+    orientation: vertical;
+    Box {
+      height-request: 360;
+      width-request: 360;
+      halign: center;
+      margin-bottom: 24;
+      orientation: vertical;
+      styles ["card"]
+
+      Adw.NavigationView nav_view {
+        Adw.NavigationPage nav_pageone {
+          title: _("Page 1");
+          Adw.StatusPage {
+            title: _("Hello 👋");
+            description: _("➡️");
+          }
+        }
+        Adw.NavigationPage nav_pagetwo {
+          title: _("Page 2");
+          Adw.StatusPage {
+            title: _("from");
+            description: _("➡️");
+          }
+        }
+        Adw.NavigationPage nav_pagethree {
+          title: _("Page 3");
+          Adw.StatusPage {
+            title: _("Workbench :)");
+            description: _("➡️");
+          }
+        }
+        Adw.NavigationPage nav_pagefour {
+          title: _("Page 4");
+          Adw.StatusPage {
+            title: _("Navigation View");
+            description: _("A page-based navigation container.");
+            Box {
+              orientation: vertical;
+              spacing: 18;
+              vexpand:true;
+              LinkButton {
+                label: _("API Reference");
+                uri: "https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.NavigationView.html";
+              }
+            }
+          }
+        }
+      }
+    }
+    Box {
+      halign: center;
+      margin-bottom: 24;
+      styles ["linked"]
+      Button previous_button {
+        label: _("Previous Page");
+        sensitive: false;
+      }
+      Button next_button {
+        label: _("Next Page");
+      }
+    }
```

Comment:
This should probably be above the view widget, and I'd suggest using `circular` & `suggested-action` styles, with symbolic icons like `go-previous-symbolic` & `go-next-symbolic`, set in `Gtk.CenterBox` with the buttons in separate sides.

For the "previous" button, should be able to just use `action-name: navigation.pop;`, while the other button would probably need a signal callback or custom action.

I'm not sure how @sonny wants this demo to look in general, but that would be a much more HIG way to do it.

Reply:
![image](https://github.com/sonnyp/Workbench/assets/96797205/d77fd735-ab87-4a59-9362-c480b0b1b111)

Looks more clean and better